sympy/series/tests/test_limits.py

```diff
@@ -493,3 +493,6 @@ def test_limit_with_Float():
 
 def test_issue_10610():
     assert limit(3**x*3**(-x - 1)*(x + 1)**2/x**2, x, oo) == S(1)/3
+
+def test_issue_6599():
+    assert(limit((n+cos(n))/n,n,oo)) == 1
```

Comment:
A space on either side of + and after each comma is expected.
No parentheses after assert are needed.